lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs

```diff
@@ -443,6 +477,11 @@ public void OnNext(ICompletedTask completedTask)
             Logger.Log(Level.Info, "Received ICompletedTask {0}, with systemState {1} in retry# {2}.", completedTask.Id, _systemState.CurrentState, _numberOfRetries);
             lock (_lock)
             {
+                if (_taskForceClosed.Contains(completedTask.Id))
```

Comment:
We set id to _taskForceClosed only in ShutDown state. IF the completed event comes later, it could be still in shutdown, could be already move to recovery, but we want to ignore it anyway to avoid double count the event.